tools/oio-bootstrap.py

```diff
@@ -426,6 +426,32 @@ def save_coverage():
     - {type: system}
 """
 
+template_beanstalk_watch = """
+host: ${IP}
+port: ${PORT}
+type: beanstalkd
+location: ${LOC}
+checks:
+    - {type: beanstalk, tube: oio}
+slots:
+    - beanstalk
```

Comment:
To remove any ambiguity, we should write beanstalk**d** everywhere.

Reply:
:+1: